README.md

```diff
@@ -54,22 +54,6 @@ Example transaction data for `abieos_json_to_bin`:
 }
 ```
 
-## Ubuntu 16.04 with gcc 8.1.0
-
-* Install these. You may have to build them yourself from source or find a PPA. Make them the default.
-  * gcc 8.1.0
-  * cmake 3.11.3
-* `sudo apt install libboost-dev libboost-date-time-dev`
-* remove this from CMakeLists.txt (2 places): `-fsanitize=address,undefined`
-
-```
-mkdir build
-cd build
-cmake ..
-make
-./test
-```
-
```

Comment:
Do we want to replace them with build instructions for supported Ubuntu versions like 22.04?

Reply:
yeah I thought that too -- really only the first half of this section was anything specific to ubuntu16. The second half:
```
mkdir build
cd build
cmake ..
make
./test
```
is pretty generic. Think it's worthwhile putting the second half back?